### DISCUSSION TITLE
Same app, different workspaces for different models?

### DISCUSSION THREAD

**Discussion by che85** [Project-MONAI/MONAILabel#1468] [Q&A] [answered]
Hello, 

I am curious about how to deploy multiple models (for different use cases - e.g. localization_spine,segmentation_vertebra). 

I have these two models currently running on the same machine on different ports with different monailabel instances, but I am wondering if it would be possible to run it on the same monailabel instance:

The following seems possible, but I am wondering why the models are sharing the same workspace.
```
monailabel start_server --app workspace/radiology --studies workspace/images --conf models "localization_spine,segmentation_vertebra"
```

Ideally, I want to have something like this:

```
monailabel start_server --app workspace/radiology --studies workspace/spine,workspace/vertebra --conf models "localization_spine,segmentation_vertebra"
```

Am I missing something? Can I configure this in the datastore? What is the quickest way to get this working?

Or do you recommend using different monailabel apps for this use case?

Thanks a lot, 
Christian

**Comment by SachidanandAlle** [comment, marked as answer]
Unfortunately, due to current design limitations, only one studies can be used... 

the initial idea for supporting multiple models per studies is.. annotators might use multiple models.. each partially contributing to whole annotation work..  at the end they will get all the labeling (multiple organs) annotation work done per image..  

i believe, in your example, you have specific samples per model only.. 
using forward proxy can be one solution but I think it's may not still be the perfect solution.

**Comment by che85** [comment]
@SachidanandAlle Why is there a dropdown for `App` selection, if you cannot serve multiple apps per monailabel server? If you don't support that functionality, maybe it shouldn't be a dropdown. 

![image](https://github.com/Project-MONAI/MONAILabel/assets/10195822/3c854750-a6f1-45cd-9dab-8ba984a6a64f)

**Reply by SachidanandAlle** [reply]
It was meant for future..  if it's bothering users, we can convert it to line edit instead of combo

**Reply by che85** [reply]
> It was meant for future.. if it's bothering users, we can convert it to line edit instead of combo

No need to. It just seemed confusing and my case is a bit moe specific, but I will probably go with running multiple monailabel instances on different ports for each app. 

THank you for your quick response.
